ref/general/microshed.adoc

```diff
@@ -61,30 +58,26 @@ The @Container annotation injects a REST _Client_ proxy of the PersonService, wh
     public static PersonService personSvc;
 
     @Test
-    public void testGetPerson() {
+    public void testGetPerson()
 
 The @RESTClient annotation invokes an HTTP POST request to the running container, which triggers the PersonService#createPerson endpoint and returns the generated ID
         Long bobId = personSvc.createPerson("Bob", 24);
 
-  Using the generated ID, invoke an HTTP GET request to read the record we just created. The JSON response will be automatically converted to a 'Person' object using JSON-B
+Using the generated ID, invoke an HTTP GET request to read the record we just created. The JSON response will be automatically converted to a 'Person' object using JSON-B
```

Comment:
why does it switch to 1st person pl? For consistency with the rest of the doc, you could stay in 2nd person. Also- avoid future tense and finish with a period

Using the generated ID, invoke an HTTP GET request to read the record you just created. The JSON response is automatically converted to a 'Person' object using JSON-B.